src/wallet/rpcwallet.cpp

```diff
@@ -2744,6 +2745,9 @@ static UniValue listunspent(const JSONRPCRequest& request)
                             {RPCResult::Type::STR, "scriptPubKey", "the script key"},
                             {RPCResult::Type::STR_AMOUNT, "amount", "the transaction output amount in " + CURRENCY_UNIT},
                             {RPCResult::Type::NUM, "confirmations", "The number of confirmations"},
+                            {RPCResult::Type::NUM, "ancestorcount", "If transaction is in the mempool, the number of in-mempool ancestor transactions (including this one)"},
+                            {RPCResult::Type::NUM, "ancestorsize", "If transaction is in the mempool, the virtual transaction size of in-mempool ancestors (including this one)"},
+                            {RPCResult::Type::STR_AMOUNT, "ancestorfees", "If transaction is in the mempool, total fees of in-mempool ancestors (including this one) with fee deltas used for mining priority"},
```

Comment:
f65a341 nit: `s/total fees/the total fees/`

while touching this help, maybe `s/the/The/` for the listunspent txid, vout, address, and scriptPubKey fields